### PR TITLE
[MFTF] Removed dependency of Media Gallery tests on ASI

### PR DESCRIPTION
**Pull request by gabrieldagama**
### Description (*)

Removed dependency of Media Gallery tests on ASI

### CODE REVIEW

**Comment by gabrieldagama** [comment]
@magento run all tests

File: MediaGalleryUi/Test/Mftf/ActionGroup/AdminMediaGalleryClickImageInGridActionGroup.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
             <description>Select image on enhanced media gallery</description>
         </annotations>
         <arguments>
-            <argument name="imageName" type="string" defaultValue="{{AdobeStockImageData.name}}"/>
+            <argument name="imageName" type="string" defaultValue=""/>
```
Comment (sivaschenko) [inline MediaGalleryUi/Test/Mftf/ActionGroup/AdminMediaGalleryClickImageInGridActionGroup.xml]: Should we just remove the defaultValue in this case?

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for the pull request @gabrieldagama ! Please see my review comments

**Comment by gabrieldagama** [comment]
@magento run all tests

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**